node/src/chain_spec/mod.rs

```diff
@@ -273,6 +273,7 @@ pub fn testnet_genesis(
             ContentConfig {
                 next_curator_group_id: 1,
                 next_channel_category_id: 1,
+                // REMARK: do not set to 0 since it will break Content Treasury Handler logic
```

Comment:
What does this mean?

Reply:
- Content treasury account Id, used for storing `bloat_bond`s (for video comments) has address `encode(MODULE_ID, 0)`
- Any other channel account has address `encode(MODULE_ID, channel_id)`

If At genensis first channel is `0` then it can be credited with bloat bonds (for example).
I realize now that this is not a 100% robust solution. So in order to address this and the next comment:
- Treasury account = `encode((MODULE_ID, "TREASURY"))`
- Any other internal channel account = `encode((MODULE_ID, "CHANNEL", channel_id))`

Reply:
in this way `channel_id` s can be started at 0 with no side effects